tests/gnrc_sock_dns/README.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+
+This folder contains a test application for RIOT's sock-based DNS client.
+
+# How to test with native
+
+Setup up a tap interface:
+
+    $ sudo ip tuntap add dev tap0 mode tap user $(id -u -n)
+    $ sudo ip a a 2001:db8::1/64 dev tap0
```

Comment:
Only adding a global address causes `dnsmasq` to send out RAs with a global address, [which is invalid](https://tools.ietf.org/html/rfc4861#section-6.1.2) and handled as such by RIOT (they are ignored). The following patch did he trick for me:

```diff
diff --git a/tests/gnrc_sock_dns/README.md b/tests/gnrc_sock_dns/README.md
index efe7e88..dcb6b7c 100644
--- a/tests/gnrc_sock_dns/README.md
+++ b/tests/gnrc_sock_dns/README.md
@@ -7,6 +7,7 @@ This folder contains a test application for RIOT's sock-based DNS client.
 Setup up a tap interface:
 
     $ sudo ip tuntap add dev tap0 mode tap user $(id -u -n)
+    $ sudo ip a a fe80::1/64 dev tap0
     $ sudo ip a a 2001:db8::1/64 dev tap0
     $ sudo ip link set up dev tap0
 
```